doc/devices/overview.rst

```diff
@@ -0,0 +1,65 @@
+Overview
+========
+
+Once the PennyLane-Qiskit plugin is installed, the following Qiskit devices
+can be accessed straightaway in PennyLane.
+
+Devices
+~~~~~~~
+
+Currently, there are three devices for different qiskit "providers":
+
+.. devicegalleryitem::
+    :name: 'qiskit.aer'
+    :description: Qiskit's staple simulator with great features such as noise models.
+    :link: aer.html
```

Comment:
These look really nice!
![image](https://user-images.githubusercontent.com/2959003/78518131-baf15c80-77fe-11ea-8310-2b8cfb6caa54.png)

We should steal this design for the PL docs, in cases where we want to have cards, but don't have any thumbnails/images.